src/index.js

```diff
@@ -0,0 +1,14 @@
+import './style.css';
+// import Leaderbord from '../modules/leaderbord.js';
+// import Selector from '../modules/selector.js';
+
+// const objLeaderbard = new Leaderbord();
+// const objSelector = new Selector();
+
+// window.addEventListener('load', () => {
+//     objLeaderbard.dispaly();
+//   });
+
+// objSelector.jsForm.addEventListener('submit', (e) => {
+//     objLeaderbard.addTeam(e);
```

Comment:
- [OPTIONAL] I'll suggest you remove this as leaving a commented piece of code in your project makes it look unprofessional and it's against the best practices. :smile: :+1:

Reply:
Thanks a lot, I will remove this line commented piece of code.